### DISCUSSION TITLE
Warning when running bitwardenrs/server:latest on Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by nilquera** [dani-garcia/vaultwarden#1280] [General]
According to the documentation, 
> The bitwardenrs/server image is also multi-arch, meaning it supports multiple CPU architectures under a single image name. 

However, after pulling bitwardenrs/server and running it, I get the following warning: 
`WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm/v7) and no specific platform was requested`
Despite the warning, the server seems to run well (i haven't tested much).

The documentation also says that Historical tags "shouldn't be used anymore", that's why I don't feel totally safe pulling the `raspberry` tag.

Should I use this raspberry tag or go ahead with the latest despite the warning message?

**Comment by jjlin** [comment]
This looks like a duplicate of #1264.

**Comment by BlackDex** [comment]
Also, this is not something we can fix on our side.
